src/anim/anim.js

```diff
@@ -717,6 +717,36 @@ Object.assign(pc, function () {
                     }
                 };
                 return new pc.AnimTarget(func, 'vector', morphInstance.morph._targets.length);
+            },
+            'materialTexture': function (node, textureName) {
+                var object = node;
+                while (object && object.constructor !== pc.Entity) {
+                    object = object.parent;
+                }
+                if (!object ||
+                    !object.model ||
+                    !object.model.model) {
+                    return null;
+                }
+                var meshInstances = object.model.meshInstances;
+                var meshInstance;
+                for (var i = 0; i < meshInstances.length; ++i) {
+                    if (meshInstances[i].node.name === node.name) {
+                        meshInstance = meshInstances[i];
+                        break;
+                    }
+                }
+                if (!meshInstance) {
+                    return null;
+                }
+                var func = function (value) {
+                    var textureAsset = pc.app.assets.get(value[0]);
```

Comment:
Don't use `pc.app`. If you don't have the application at hand, you can use `Application.getApplication()`. Remember, `pc.` is now banned in internal engine code. 😄